src/main/java/dev/jbang/util/Util.java

```diff
@@ -67,6 +69,7 @@
 			"^.*(public\\s+static|static\\s+public)\\s+void\\s+main\\s*\\(.*",
 			Pattern.MULTILINE);
 
+	private static final List<String> EXTENSIONS = asList(".java", ".jsh", ".kt");
```

Comment:
".kts" is what kscript defaults to so we so should consider adding that too so you can do:

" jbang https://github.com/holgerbrandl/kscript/blob/master/examples/classpath_example.kts"

Reply:
ah - seem there is a little more to support .kts ...at least renaming from .kts to .kt isn't enough :)

Reply:
.kts would be using kotlinc to run ...so slightly different. lets make that a future idea.